style.css

```diff
@@ -74,27 +73,318 @@ header {
   padding: 10px;
   height: 20px;
   flex-wrap: nowrap;
-  margin-top: -20px;
+  margin-top: -25px;
+}
+
+.frame,
+li {
+  padding: 2px;
+}
+
+ul li {
+  list-style: none;
+  margin: 5px;
 }
 
 .icon li {
   padding: 10px;
 }
 
-.contact {
-  padding: 0 0 0 20px;
-  font-family: 'Poppins', sans-serif;
+a {
+  text-decoration: none;
+}
+
+.connect,
+a {
+  color: #7f8cff;
+}
+
+p a {
+  color: #7f8cff;
+}
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 4em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 8px;
+  min-height: 75vh;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.pic1 img {
+  width: 100%;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  padding: 0%;
+  margin-left: -5px;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  margin-top: 0;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin-top: 0;
+  margin-left: -5px;
+}
+
+.language {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.html,
+.css,
+.java {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 4px 12px;
+}
+
+.card-btn {
+  text-align: left;
+  background-color: white;
+  color: #396df2;
+  width: fit-content;
+  border-radius: 8px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  padding: 12px;
+}
+
+.card-btn:hover {
+  background-color: #2230d2;
+  color: white;
+}
+
+.title {
+  width: 34%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.bottom {
+  margin: 0 0 16px -25px;
+}
+
+.bio {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  padding: 0 20px;
+  margin-left: 15px;
+  margin-top: -30px;
+}
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
- [ ] As per requested by the previous reviewer, the section **about myself** has the top right corner curved. It's important to take notice to little details in design because it will set you apart from many developers 📱 You can set the **border-radius** to achieve that and you have to change the background color so you can see the changes ⚪ 

Your design 👇🏽 

![Screenshot from 2023-07-08 22-55-01](https://github.com/ali-techworld/Portfolio-Setup/assets/10439283/6bcd5cc8-2d7c-41e1-b8bd-a97e9a42fd4a)

Design from Figma 👇🏽 

![Screenshot from 2023-07-08 23-01-26](https://github.com/ali-techworld/Portfolio-Setup/assets/10439283/349f5c67-41cb-47fe-ac4d-10f8156c7833)